### DISCUSSION TITLE
UCAN Discord, github bots

### DISCUSSION THREAD

**Discussion by bmann** [ucan-wg/spec#100] [General]
We'll want to link to the UCAN Discord prominently in a couple of places I think https://discord.gg/U8sXZNZ4FB, including the community website at https://github.com/ucan-wg/ucan-check (which we should rename, too!)

For bots, @walkah can you help get Github bots setup for the Discord?

**Comment by walkah** [comment]
Clearing out old github notifications - this is done... is there a "mark as resolved" option for discussions?
